source/install/upgrade_from_40.rst

```diff
@@ -46,7 +46,7 @@ Upgrading on RHEL and CentOS
 |tip.run-all.root|
 
 1. Stop the ``mongod`` instance: :bash:`service mongod stop`
-#. Remove |PSMDB| |prev-version| packages: :bash:`yum remove Percona-Server-MongoDB-36*`
+#. Remove |PSMDB| |prev-version| packages: :bash:`yum remove Percona-Server-MongoDB-40*`
```

Comment:
same for rpm (no 40 in package name), but for rpm they are also all lowercase, so `Percona-Server-MongoDB-40*` should be `percona-server-mongodb*`